### PR TITLE
Change "setup" to "set up" in source language

### PR DESCRIPTION
**Pull request by systemcrash**

### CODE REVIEW

File: src/strings/en-gb.json [hunk 1]
```diff
@@ -1196,7 +1196,7 @@
     "HeaderSpecialEpisodeInfo": "Special Episode Info",
     "HeaderSortOrder": "Sort Order",
     "HeaderSortBy": "Sort By",
-    "HeaderSetupLibrary": "Setup your media libraries",
+    "HeaderSetupLibrary": "Set up your media libraries",
```
Comment (nielsvanvelzen) [inline src/strings/en-gb.json:1199]: Changes to translations should be made directly in https://translate.jellyfin.org/

Comment (systemcrash) [reply src/strings/en-gb.json:1199]: Absolutely right. Should. I tried and it was uppgefuckad, so here you have it instead.

Comment (nielsvanvelzen) [reply src/strings/en-gb.json:1199]: Not sure what you mean, translations should not be updated via pull requests as this often breaks our translation tool. Only edits in `en-us.json` (the source language) can be done in a pull request.

Comment (systemcrash) [reply src/strings/en-gb.json:1199]: Ah well. 50% of the way there 😅

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**